app/controllers.js

```diff
@@ -87,39 +242,77 @@ Controller.prototype.handleApiEditProject = function(req, res) {
             }
         });
     }
-}
+};
 
 // All the data as JSON
 Controller.prototype.handleApi = function(req, res) {
     res.json(projectCache.getAll());
-}
+};
 
 // Project info
 Controller.prototype.handleProjectIdSlug = function(req, res) {
     var data = projectCache.getById(req.params.id);
     res.render('project', {
         "data": data,
-        "phase_order": phase_order,
+        "phase_order": phase_order
     });
-}
+};
 
 // Prototype version of project info 
 Controller.prototype.handleSlugPrototype = function(req, res) {
     var data = projectCache.getById(req.params.id);
     if (typeof data.prototype == 'undefined') {
         res.render('no-prototype', {
-            "data": data,
+            "data": data
         });
     } else {
         res.redirect(data.prototype);
     }
-}
+};
 
 // Add project form
 Controller.prototype.handleAddProject = function(req, res) {
     var id = req.params.id;
     res.render('add-project');
-}
+};
+
+/**
+ * Render Edit team memberspage
+ *
+ * @param req
+ * @param res
+ */
+Controller.prototype.handleEditOurTeam = function (req, res) {
+    var id = req.params.id;
+
+    var project = projectCache.getAll()[id];
+
+    res.render('edit_our_team', {
```

Comment:
Can we change `edit_our_team` to `edit-our-team` so it's similar to the other views?